app/client/src/features/events/EventPre.tsx

```diff
@@ -0,0 +1,77 @@
+import { useRouter } from 'next/router';
+import { PreloadedQuery, usePreloadedQuery, useQueryLoader } from 'react-relay';
+import { CountdownWrapper } from '@local/components/Countdown';
+import { EventLiveQuery } from '@local/__generated__/EventLiveQuery.graphql';
+import { EVENT_LIVE_QUERY } from './EventLive';
+import { BroadcastMessageList } from './BroadcastMessages/BroadcastMessageList/BroadcastMessageList';
+import React from 'react';
+import { EventContext } from './EventContext';
+
+const styles = {
+    justifyCenter: {
+        display: 'flex',
+        justifyContent: 'center',
+        width: '100%',
+    },
+    list: {
+        display: 'flex',
+        justifyContent: 'center',
+        width: '100%',
+        paddingTop: '40px',
+    },
+};
+
+export interface PreloadedEventLiveProps {
+    eventLiveQueryRef: PreloadedQuery<EventLiveQuery>;
+}
+
+export function EventPre({ eventLiveQueryRef }: PreloadedEventLiveProps) {
+    const router = useRouter();
+    const { node } = usePreloadedQuery(EVENT_LIVE_QUERY, eventLiveQueryRef);
```

Comment:
Not sure if you saw my earlier message about this, but please make sure to create a new query for this since we do not need all the same data that the `EVENT_LIVE_QUERY` has. I suggest making a `PRE_EVENT_QUERY`.